boards/raspberry-pi-4/ubuntu_server_20.04_arm64.json

```diff
@@ -0,0 +1,46 @@
+{
+  "variables": {},
+  "builders": [{
+    "type": "arm",
+    "file_urls" : ["http://cdimage.ubuntu.com/releases/20.04.1/release/ubuntu-20.04.1-preinstalled-server-arm64+raspi.img.xz"],
+    "file_checksum_url": "http://cdimage.ubuntu.com/releases/20.04.1/release/SHA256SUMS",
+    "file_checksum_type": "sha256",
+    "file_target_extension": "xz",
+    "file_unarchive_cmd": ["xz", "--decompress", "$ARCHIVE_PATH"],
+    "image_build_method": "reuse",
+    "image_path": "ubuntu-20.04.img",
+    "image_size": "3.1G",
+    "image_type": "dos",
+    "image_partitions": [
+      {
+        "name": "boot",
+        "type": "c",
+        "start_sector": "2048",
+        "filesystem": "fat",
+        "size": "256M",
+        "mountpoint": "/boot"
+      },
+      {
+        "name": "root",
+        "type": "83",
+        "start_sector": "526336",
+        "filesystem": "ext4",
+        "size": "2.8G",
+        "mountpoint": "/"
+      }
+
+    ],
+    "image_chroot_env": ["PATH=/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:/sbin"],
+    "qemu_binary_source_path": "/usr/bin/qemu-arm-static",
+    "qemu_binary_destination_path": "/usr/bin/qemu-arm-static"
```

Comment:
And this:
`"qemu_binary_destination_path": "/usr/bin/qemu-aarch64-static"`

Maybe also adding this board to the matrix in https://github.com/mkaczanowski/packer-builder-arm/blob/master/.github/workflows/all-boards.yml would be beneficial to test and catch those things.

Reply:
thanks.

the build is failing with:
arm: chroot: failed to run command ‘/bin/sh’: No such file or directory

I'll check that one in the evening

Reply:
odd. the build ran fine on my box... will try it again a bit later today...

Reply:
it might have been some transient issue. Now it looks all fine:
![image](https://user-images.githubusercontent.com/4912567/89692574-61b04b80-d90c-11ea-8ee7-19a48861ab30.png)

Reply:
thanks @dbast @bradwood !